core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -77,29 +77,46 @@
             </span>
           </a>
           <ul ngbDropdownMenu
-              class="dropdown-menu dropdown-menu oppia-navbar-dropdown"
+              class="dropdown-menu oppia-navbar-dropdown classroom-dropdown-menu"
               (mouseover)="openSubmenu($event, 'classroomMenu')"
               (mouseleave)="closeSubmenuIfNotMobile($event)"
               role="menu"
               autofocus>
             <li class="nav-item">
-              <a (keydown)="onMenuKeypress($event, 'classroomMenu', {shiftTab: ACTION_CLOSE})"
-                 (click)="navigateToClassroomPage('/learn/math')"
-                 class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link"
-                 href="#"
-                 [innerHTML]="'I18N_TOPNAV_CLASSROOM_BASIC_MATHS' | translate">
-              </a>
+              <div class="nav-item-main d-flex justify-content-between">
+                <div class="nav-item-left d-flex flex-column">
+                  <div class="nav-item-top d-flex justify-content-between">
+                    <div class="nav-content d-flex flex-column">
+                      <a (keydown)="onMenuKeypress($event, 'classroomMenu', {shiftTab: ACTION_CLOSE})" (click)="navigateToClassroomPage('/learn/math')" class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link nav-item-top-link" href="#" [innerHTML]="'I18N_TOPNAV_CLASSROOM_BASIC_MATHS' | translate"></a>
+                      <div class="des">
+                        Beginner friendly lessons to help you get started in math.
```

Comment:
Can we add an I18n key for `Beginner friendly lessons to help you get started in math.` and all the text elsewhere in the html? We use I18n keys so that the site can be available in different language, so, we do not prefer using text directly.

Reply:
Done